methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+def ending(number)
+  lyrics = <<lyrics
+#{number} bottle of beer on the wall, #{number} bottle of beer.
+Take one down and pass it around, no more bottles of beer on the wall.\n
+No more bottles of beer on the wall, no more bottles of beer.
+Go to the store and buy some more, 99 bottles of beer on the wall.
+lyrics
+  print lyrics
+end
+
+def single_bottle(number)
+  lyrics = <<lyrics
+#{number} bottles of beer on the wall, #{number} bottles of beer.
```

Comment:
Lots of duplication here...  DRY up your code by making a method that takes number of bottles and returns that number with bottles or bottle?